packages/lms/src/lib/components/ui/Icon.svelte

```diff
@@ -0,0 +1,6 @@
+<script lang="ts">
+  export let iconName: string;
+  import 'material-icons/iconfont/material-icons.css';
+</script>
+
+<span class="material-icons">{iconName}</span>
```

Comment:
```suggestion
<span class="material-icons" {...$restProps}>{iconName}</span>
```
I'm a huge fan of the `$restProps` feature Svelte offers. If you suddenly need some extra styling applied directly to the `span` you can then use the in-line `style` attribute, for example. Or if you need to add eventListeners you can do that directly without having to edit the Icon component.